src/allmydata/storage_client.py

```diff
@@ -844,10 +843,10 @@ def get_available_space(self):
         version = self.get_version()
         if version is None:
             return None
-        protocol_v1_version = version.get('http://allmydata.org/tahoe/protocols/storage/v1', UnicodeKeyDict())
-        available_space = protocol_v1_version.get('available-space')
+        protocol_v1_version = version.get(b'http://allmydata.org/tahoe/protocols/storage/v1', BytesKeyDict())
```

Comment:
Didn't this start out as JSON? Not sure I understand why the keys should be bytes...

Reply:
This comes in via remote get_version() Foolscap call, where default strings were bytes, so for wire compatibility it continues to be bytes.